### ISSUE TITLE
Gantry5 replaced logo shows only in Safari browser

### ISSUE THREAD

**Issue by Attis35** [gantry/gantry5#519]
Hello,

I've made a website with gantry5 and I uploaded a .png logo to the images folder. Under the layouts tab I've added the Logo particle and set my previously uploaded image as a logo but the logo only visible with Safari and it doesn't work with other browsers.

Could you please help me?
Thank you!

**Comment by mahagr** [comment]
Can you check the URL in the document source if it's correct. It should be absolute URL from site root.

**Comment by w00fz** [comment]
A link would be helpful here, not enough informations to figure out what the issue could be.
Or if you feel like, jump on Gitter and someone will have a closer look at it!

[![Join the chat at https://gitter.im/gantry/gantry5](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/gantry/gantry5?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)

**Comment by Attis35** [comment]
Sorry for late answer, I was on vacation. Meanwhile I've solved the problem. I've added the logo in the GUI within Template Manager, Layouts -> Logo -> Image. I guess the problem was the size of the png (it was too small). When I replaced it with a jpg, it worked, than I made another (larger) png which also worked for me. It's weird, that Safari and Iphone6 displayed it perfectly, but no luck with IE and FF.
